new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/FileUtils.java

```diff
@@ -55,4 +63,55 @@ public static boolean doesFileExist(String fileName) {
     }
     return false;
   }
+  
+  /**
+   * Remove the given directory and its contents.
+   *
+   * @param dir the directory to be cleaned up
+   */
+  public static void cleanupDirectory(String dir) throws IOException {
+    File file = new File(dir);
+    logger.info("Cleaning up directory " + dir);
+    if (!file.exists()) {
+      // nothing to do
+      return;
+    }
+
+    assertThat(file.isDirectory())
```

Comment:
this feel like it could be a simple check and a throw IllegalArgsException - assertions are really for tests

Reply:
done.